website/docs/r/nrql_drop_rule.html.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: "newrelic"
+page_title: "New Relic: newrelic_nrql_drop_rule"
+sidebar_current: "docs-newrelic-resource-nrql-drop-rule"
+description: |-
+  Create and manage NRQL Drop Rules.
+---
+
+# Resource: newrelic\_nrql\_drop\_rule
+
+Use this resource to create, and delete New Relic NRQL Drop Rules.
+
+-> **IMPORTANT!** Version 2.0.0 of the New Relic Terraform Provider introduces some [additional requirements](/docs/providers/newrelic/index.html) for configuring the provider.
+<br><br>
+Before upgrading to version 2.0.0 or later, it is recommended to upgrade to the most recent 1.x version of the provider and ensure that your environment successfully runs `terraform plan` without unexpected changes.
+
+## Example Usage
+
+```hcl
+resource "newrelic_nrql_drop_rule" "foo" {
+  account_id = 12345
+  description = "Example NRQL drop rule description"
+  action = "drop_data"
+  nrql = "SELECT * FROM Log WHERE container_name = 'noise'"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+  * `account_id` - (Required) Account where the drop rule will be put.
+  * `description` - (Required) The description of the drop rule.
```

Comment:
description is optional.  Should mention that changing it forces a new rule to be created.